docs/TektonResult.md

```diff
@@ -148,3 +148,22 @@ stringData:
   S3_MULTI_PART_SIZE: "5242880"
 ```
 
+
+### GCS specific Property
+The follow keys are needed for GCS:
+* `gcs_creds_secret_name`
+* `gcc_creds_secret_key`
+* `gcs_bucket_name`
+
+We need to create a secret with google application creds for a bucket `foo-bar` like below:
+
+```
+kubectl create secret generic gcs-credentials --from-file=creds.json
+```
+
+In the above example, our properties are:
+```
+gcs_creds_secret_name: gcs-credentials
+gcc_creds_secret_key: creds.json
+gcs_bucket_name: foo-bar
+```
```

Comment:
@khrm Where do we add these properties? under `TektonResult.spec`, if so please include those on that section.
From where do we get the `creds.json`, Can you please explain about that?

Reply:
Sure. I would add a link to GSC.

Reply:
I think it's clear now. I also added clarification about creds.json.